lib/kimono_labs_client/web_client.rb

```diff
@@ -0,0 +1,31 @@
+module KimonoLabsClient
+  # Web client for fetching a json endpoint's content via http.
+  class WebClient
+    attr_reader :api_key
+
+    # initializer
+    def initialize(api_key)
+      fail 'API key must be provided!' if api_key.nil?
```

Comment:
Would you move the fail then?  It feels weird to me to assign the attribute first and then fail if it's not set.  Right now I have this:

    def initialize(api_key)
      @api_key = api_key
      @http = http

      fail 'API key must be provided!' unless key?
    end

    def key?
      !@api_key.blank?
    end